zipline/pipeline/loaders/blaze/core.py

```diff
@@ -595,9 +585,11 @@ def from_blaze(expr,
     ds = new_dataset(dataset_expr, deltas)
     # Register our new dataset with the loader.
     (loader if loader is not None else global_loader)[ds] = ExprData(
-        dataset_expr,
-        deltas,
-        resources,
```

Comment:
we are binding the resources into the expression like we do with the earnings loader